microsoft-edge/devtools-guide-chromium/speed/get-started.md

```diff
@@ -22,440 +22,480 @@ ms.date: 05/04/2021
-1. Audit the page again.
+    This webpage is just a demo and a `for` loop was introduced to slow down the code. In practice, there are many ways for a JavaScript file like this one to cause slow page loads, such as: accessing the DOM too frequently, running heavy computation tasks, using large JavaScript libraries or frameworks.
 
-   ![An Audits report after removing unnecessary JavaScript work](../media/speed-glitch-tony-remix-updated-4-audits-performance.msft.png)
+1. Delete the `for` loop at the beginning of the `initRatings` function, save your changes, and reload the page in Microsoft Edge.
 
-Looks like that last change caused a massive jump in performance!
+1. Run a new audit in the **Lighthouse** tool to see how your changes affect the load performance:
 
-This section provided a rather brief introduction to the Performance tool.  To learn more about how to analyze page performance, see [Performance features reference)](../evaluate-performance/reference.md).
+    ![Your new report, showing a much better score](./images/report-after-for-loop.png)
 
-#### Doing less main thread work in the real world
+    Your score went up to 92, and the **Time to interactive** and **Total Blocking Time** metrics are down to 1.3 seconds and 0 seconds respectively.
 
-The **Performance** tool is the most common way to understand what activity your site does as it loads, and to find ways to remove unnecessary activity.
+To learn more about how to analyze page performance, see [Performance features reference)](../evaluate-performance/reference.md).
 
-If you prefer an approach that feels more like `console.log()`, the [User Timing API](https://developer.mozilla.org/docs/Web/API/User_Timing_API) enables you to arbitrarily mark up certain phases of your app lifecycle, in order to track how long each of those phases takes.
+###### Doing less main thread work in the real world
 
+The **Performance** tool is the most common way to understand what activity your webpage does as it loads, and to find ways to remove unnecessary activity.
 
-<!-- ====================================================================== -->
-## Summary
+If you prefer an approach that feels more like `console.log()`, the [User Timing API](https://developer.mozilla.org/docs/Web/API/User_Timing_API) enables you to arbitrarily mark up certain phases of your app lifecycle, in order to track how long each of those phases takes.
 
-*  Whenever you set out to optimize the load performance of a site, always start with an audit.  The audit establishes a baseline, and gives you tips on how to improve.
-*  Make one change at a time, and audit the webpage after each change in order to display how that isolated change affects performance.
+Webpages often load slowly when they use JavaScript for generating most of their user interfaces. Moving the code that creates your webpage's user interface to the server-side and delivering it as HTML and CSS to the browser can improve your webpage's performance.
```

Comment:
```suggestion
Webpages often load slowly when they use JavaScript for generating most of their user interfaces.  To improve your webpage's performance, consider moving the code that creates your webpage's user interface to the server side, delivering that code to the browser as HTML and CSS.
```
hyphen would be right if:   server-side code
forming an adjective phrase modifying a following word (that is an alt'v)